vault/data_source_aws_access_credentials.go

```diff
@@ -118,13 +118,27 @@ func awsAccessCredentialsDataSourceRead(d *schema.ResourceData, meta interface{}
 	}
 
 	d.SetId(secret.LeaseID)
-	d.Set("access_key", secret.Data["access_key"])
-	d.Set("secret_key", secret.Data["secret_key"])
-	d.Set("security_token", secret.Data["security_token"])
-	d.Set("lease_id", secret.LeaseID)
-	d.Set("lease_duration", secret.LeaseDuration)
-	d.Set("lease_start_time", time.Now().Format(time.RFC3339))
-	d.Set("lease_renewable", secret.Renewable)
+	if err := d.Set("access_key", secret.Data["access_key"]); err != nil {
```

Comment:
While I normally would 👍 the error handing, I've seen error checking on `Set()` only occasionally, and even the "how to extend TF" docs omit it. I've wondered about what the correct pattern is. Thoughts on this @catsby ?

Reply:
Ah! Thank you! I'll update that.